assets/xml/objects/object_yb.xml

```diff
@@ -1,57 +1,58 @@
+        <DList Name="gYbRightFoodDL" Offset="0x3900" />
+        <DList Name="gYbLeftThighDL" Offset="0x3BC8" />
+        <DList Name="gYbLeftCalfDL" Offset="0x3D68" />
+        <DList Name="gYbLeftFootDL" Offset="0x3EB8" />
+        <DList Name="gYbEndDisplayList1DL" Offset="0x4108" />
+        <DList Name="gYbUpperTorsoDL" Offset="0x4110" />
+        <DList Name="gYbHeadDL" Offset="0x4310" />
+        <DList Name="gYbPonyTailDL" Offset="0x47F8" />
+        <DList Name="gYbEndDisplayList2DL" Offset="0x4970" />
+        <DList Name="gYbRightUpperArmDL" Offset="0x4978" />
+        <DList Name="gYbRightForArmDL" Offset="0x4AA0" />
+        <DList Name="gYbRightHandDL" Offset="0x4BC0" />
+        <DList Name="gYbLeftUpperArmDL" Offset="0x4D98" />
+        <DList Name="gYbLeftForArmDL" Offset="0x4EC0" />
+        <DList Name="gYbLeftHandDL" Offset="0x4FE0" />
+        <Texture Name="gYbTLUT" OutName="yb_tlut" Format="rgba16" Width="16" Height="16" Offset="0x51B8" />
+        <Texture Name="gYbForheadBlotchTex" OutName="yb_forhead_blotch" Format="ci8" Width="8" Height="8" Offset="0x53B8" />
+        <Texture Name="gYbPonyTailTex" OutName="yb_ponytail" Format="ci8" Width="8" Height="16" Offset="0x53F8" />
+        <Texture Name="gYbFingersTex" OutName="yb_fingers" Format="ci8" Width="16" Height="16" Offset="0x5478" />
+        <Texture Name="gYbToesTex" OutName="yb_toes" Format="ci8" Width="16" Height="16" Offset="0x5578" />
+        <Texture Name="gYbHairLineTex" OutName="yb_hairline" Format="ci8" Width="16" Height="16" Offset="0x5678" />
+        <Texture Name="gYbBellybuttonTex" OutName="yb_bellybutton" Format="ci8" Width="8" Height="16" Offset="0x5778" />
+        <Texture Name="gYbNoseTex" OutName="yb_nose" Format="ci8" Width="16" Height="16" Offset="0x57F8" />
+        <Texture Name="gYbEyeTex" OutName="yb_eye" Format="ci8" Width="32" Height="32" Offset="0x58F8" />
+        <Texture Name="gYbPantsTex" OutName="yb_pants" Format="i8" Width="16" Height="16" Offset="0x5CF8" />
+        <Limb Name="gYbRoot1Limb" Type="Standard" Offset="0x5DF8" />
+        <Limb Name="gYbPantsLimb" Type="Standard" Offset="0x5E04" />
+        <Limb Name="gYbRoot2Limb" Type="Standard" Offset="0x5E10" />
+        <Limb Name="gYbRightThighLimb" Type="Standard" Offset="0x5E1C" />
+        <Limb Name="gYbRightCalfLimb" Type="Standard" Offset="0x5E28" />
+        <Limb Name="gYbRightFootLimb" Type="Standard" Offset="0x5E34" />
+        <Limb Name="gYbLeftThighLimb" Type="Standard" Offset="0x5E40" />
+        <Limb Name="gYbLeftCalfLimb" Type="Standard" Offset="0x5E4C" />
+        <Limb Name="gYbLeftFootLimb" Type="Standard" Offset="0x5E58" />
+        <Limb Name="gYbEmpty1Limb" Type="Standard" Offset="0x5E64" />
+        <Limb Name="gYbHeadLimb" Type="Standard" Offset="0x5E70" />
+        <Limb Name="gYbPonyTailLimb" Type="Standard" Offset="0x5E7C" />
+        <Limb Name="gYbEmpty5E88Limb" Type="Standard" Offset="0x5E88" />
```

Comment:
For the other two EmptyLimbs, you have `Empty(1/2)Limb` while here you have the offset. Could you pick one style.

Reply:
Also is it possible these are used as root limbs, but instead of a NULL DList they just used an Empty DList for some reason.